packages/functional-tests/tests/misc/authClientV2.spec.ts

```diff
@@ -44,20 +30,13 @@ test.describe('auth-client-tests', () => {
 
   test.beforeEach(({}, { project }) => {
     test.skip(project.name === 'production');
-
-    email = `signin${crypto.randomBytes(8).toString('hex')}@restmail.net`;
-    password = `${crypto.randomBytes(10).toString('hex')}`;
-  });
-
-  test.afterEach(async ({ target }) => {
-    const newCreds = await curClient?.signIn(email, password, { keys: true });
```

Comment:
In the email fixture, we don't use the signin option `{ keys: true }`. Does this need to be accommodated?

Reply:
No this does not need to be accommodated. For the majority of functional tests, keys won't be needed on sign in. We only need to do this if we have a test scenario where the user needs to derive their private key for end to end encryption purposes. This happens when syncing data in Firefox, but for many of the tests we have, like simply logging into your account, requesting keys via the `{keys:true}` option is unnecessary.